cmd/celestia-appd/cmd/gas_price_updater.go

```diff
@@ -0,0 +1,70 @@
+package cmd
```

Comment:
is this the direction we're taking? updating all the fields, that need changes, using the binary instead of asking validator operators to run a `sed`?

Reply:
Mus proposed having a binary change to update it (I gather under the assumption that operators have a tendency to bump binaries more than modifying the config)